### PR TITLE
chore: changed `dateFormat` for Timestamp value

### LINKED ISSUE #2858: [UI] Timestamp value overlaps with dropdown field icon within Topic/Messages
**Describe the bug**
The selected value of Timestamp overlaps with dropdown field icon within Topic/Messages

**Set up**
https://www.kafka-ui.provectus.io/

**Steps to Reproduce**

1. Navigate to Topic/Messages tab
2. Select Timestamp for Seek Type field
3. Choose the date for Timestamp filed

**Expected behavior**
Dropdown filed icon should not overlap with selected value

**Screenshots**
timestamp value

### PR DESCRIPTION
**Pull request by Pranav-yadav**
- [NA] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

[UI] Changed `dateFormat` for Timestamp value
- in Topic > Messages > Filters
- closes #2858

**Is there anything you'd like reviewers to focus on?**
`dateFormat` suggested by @Haarolean here https://github.com/provectus/kafka-ui/issues/2858#issuecomment-1302457968

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [NA] No need to
- [NA] Manually (please, describe, if necessary)
- [x] Unit checks
- [x] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**
# 🐨

### CODE REVIEW

**Comment by Pranav-yadav** [comment]
@Haarolean shall I update the same in `MessagesV2` R.FC as well?
https://github.com/provectus/kafka-ui/blob/1b1caddd310d3c92c8f3430b991cc48efef469ff/kafka-ui-react-app/src/components/Topics/Topic/MessagesV2/FiltersBar/Form.tsx#L127

**Comment by Haarolean** [comment]
> @Haarolean shall I update the same in `MessagesV2` R.FC as well?
> 
> https://github.com/provectus/kafka-ui/blob/1b1caddd310d3c92c8f3430b991cc48efef469ff/kafka-ui-react-app/src/components/Topics/Topic/MessagesV2/FiltersBar/Form.tsx#L127

methinks it's aight to keep it as it is for v2, there's a different layout.

**Review by Mgrdich — APPROVED** [review]

**Review by armenuikafka — CHANGES_REQUESTED** [review]
seems timestamp is broken

https://user-images.githubusercontent.com/104780608/203734385-2dfe40b2-c475-4496-87fd-65ce94447dac.mov

**Comment by Pranav-yadav** [comment]
> seems timestamp is broken

@armenuikafka I'll first merge the `master`, then will investigate... 🤔

**Comment by Pranav-yadav** [comment]
### Date Format

This PR introduces only _a single line_ of change in `dateFormat` prop used by `kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/Filters.tsx` component

`kafka-ui` uses [`react-datepicker`](https://www.npmjs.com/package/react-datepicker) which uses [`date-fns`](https://www.npmjs.com/package/date-fns) and hence, uses following Date Formats specs:
https://date-fns.org/v2.0.0-alpha.18/docs/format

and the current (_change_) `dateFormat` i.e. `MMM D, yyyy HH:mm` is <ins>**_valid_** as per the specs</ins>

@armenuikafka 

---
Cc: @Mgrdich , @Haarolean 
Since, I don't have a proper local setup for `kafka-ui` I'm unable to further investigate.. lmk if anything needs to be changed.

File: kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/Filters.tsx [hunk 1]
```diff
@@ -428,7 +428,7 @@ const Filters: React.FC<FiltersProps> = ({
                   onChange={(date: Date | null) => setTimestamp(date)}
                   showTimeInput
                   timeInputLabel="Time:"
-                  dateFormat="MMMM d, yyyy HH:mm"
+                  dateFormat="MMM D, YYYY HH:mm"
```
Comment (Pranav-yadav) [inline kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/Filters.tsx:431]:
```suggestion
                  dateFormat="MMM d, yyyy HH:mm"
```

**Review by armenuikafka — APPROVED** [review]
overlapping issue fixed !

**Pull request merged**